tests/test_packageAlerts.py

```diff
@@ -153,6 +204,49 @@ def setUp(self):
         self.cutoutWcs.wcs.cd = self.exposure.getWcs().getCdMatrix()
         self.cutoutWcs.wcs.ctype = ["RA---TAN", "DEC--TAN"]
 
+    def _deserialize_alert(self, alert_bytes):
```

Comment:
I think this, and the `_deserialize_confluent_wire_header` should both be free functions here: they don't need to live in the test class at all.

Reply:
I've removed `_deserialized_confluent_wire_header` because its not needed if we aren't doing the header version sanity check.